docs/repositories.md

```diff
@@ -174,6 +174,18 @@ poetry source add --priority=primary PyPI
 
 This way, the priority of PyPI can be set in a fine-granular way.
 
+The equivalent specification in `pyproject.toml` is:
+
+```toml
+[[tool.poetry.source]]
+name = "pypi"
+priority = "supplemental"
```

Comment:
```suggestion
priority = "primary"
```

This should be "primary" to be equivalent to the previous command, shouldn't it?

Reply:
True.  I'll change it.